src/component/Main/Main.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import style from './Main.module.css';
+import MovieItem from './MovieItem/MovieItem';
+
+const Main = () => {
+  return (
+    <main className={style.Main}>
```

Comment:
Don't you forget to create this panel 
![image](https://user-images.githubusercontent.com/45543713/95174957-6cc81080-07c3-11eb-845b-a83088c9d384.png)
?